src/python/README.md

```diff
@@ -0,0 +1,8 @@
+# 增加Cypher查询
```

Comment:
README.md is supposed to tell people what this module does. You can write these contents in the commit log.